src/server/trpc/router/bribes.ts

```diff
@@ -1,11 +1,19 @@
 //import { addLogin, readLoginList } from "utils/database/loginID.db";
+import { TRPCError } from "@trpc/server";
+import { type Session } from "next-auth";
 import { readOneBribefile } from "utils/database/bribefile.db";
-import { z } from "zod";
 
 import { router, publicProcedure } from "../trpc";
 
 export const bribesRouter = router({
-  list: publicProcedure.query(async () => {
+  list: publicProcedure.query(async ({ ctx }) => {
+    // we do not want this protected, but checking for authenticated user as a test to make sure this works
```

Comment:
I used this call as a test to make sure authentication works. We can remove in this PR or later.

Reply:
We can leave this in, as we need to add writing endpoints to the bribes router. We then can move the session query down to the mutate section.